source/DemoView.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2017 Haiku, Inc.
+ * Distributed under the terms of the MIT License.
+ *
+ * Author:
+ *		Marcin 'Shard' Konicki
+ *		Owen Pan <owenpiano@gmail.com>
+ */
+
+#ifndef _DEMO_VIEW_H
```

Comment:
Oops. I just copied the format from StripeView.h and replaced STRIPE with DEMO. I will fix it.